cli/lib/kontena/cli/nodes/show_command.rb

```diff
@@ -4,16 +4,16 @@ class ShowCommand < Kontena::Command
     include Kontena::Cli::GridOptions
     include Kontena::Cli::BytesHelper
 
-    parameter "NODE_ID", "Node id"
+    parameter "NODE", "Node"
 
     def execute
       require_api_url
       require_current_grid
       token = require_token
 
-      node = client(token).get("nodes/#{current_grid}/#{node_id}")
-      puts "#{node['name']}:"
-      puts "  id: #{node['id']}"
+      node = client(token).get("nodes/#{current_grid}/#{self.node}")
+      puts "#{node['id']}:"
```

Comment:
Also changed this, for consistency with other commands like `kontena service show`...